tests/e2e/tests/regression/insights/feature-flag.e2e.ts

```diff
@@ -0,0 +1,150 @@
+import { modifyFeaturesConfigJson, refreshFeaturesTestData, updateControlNumber } from '../../../helpers/insights';
+import { Common } from '../../../helpers/common';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const browserPage = new BrowserPage();
+const settingsPage = new SettingsPage();
+
+const featuresConfigTable = 'features_config';
+const redisVersionRecom = RecommendationIds.redisVersion;
+const pathes = {
+    defaultRemote: path.join('.', 'test-data', 'features-configs', 'insights-default-remote.json'),
+    invalidConfig: path.join('.', 'test-data', 'features-configs', 'insights-invalid.json'),
+    validConfig: path.join('.', 'test-data', 'features-configs', 'insights-valid.json'),
+    analyticsConfig: path.join('.', 'test-data', 'features-configs', 'insights-analytics-filter-off.json'),
+    buildTypeConfig: path.join('.', 'test-data', 'features-configs', 'insights-build-type-filter.json'),
+    flagOffConfig: path.join('.', 'test-data', 'features-configs', 'insights-flag-off.json'),
+    dockerConfig: path.join('.', 'test-data', 'features-configs', 'insights-docker-build.json'),
+    electronConfig: path.join('.', 'test-data', 'features-configs', 'insights-electron-build.json')
+};
+
+fixture `Feature flag`
+    .meta({ type: 'regression', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneV5Config, ossStandaloneV5Config.databaseName);
+        await refreshFeaturesTestData();
+    })
+    .afterEach(async() => {
+        // Delete database
+        await deleteStandaloneDatabaseApi(ossStandaloneV5Config);
+        await refreshFeaturesTestData();
+    });
+test('Verify that default config applied when remote config version is lower', async t => {
+    await updateControlNumber(19.2);
+
+    const featureVersion = await JSON.parse(await getColumnValueFromTableInDB(featuresConfigTable, 'data')).version;
+
+    await t.expect(featureVersion).eql(1, 'Config with lowest version applied');
+    await t.expect(browserPage.InsightsPanel.insightsBtn.exists).notOk('Insights panel displayed when disabled in default config');
+});
+test('Verify that invaid remote config not applied even if its version is higher than in the default config', async t => {
+    // Update remote config .json to invalid
+    await modifyFeaturesConfigJson(pathes.invalidConfig);
+    await updateControlNumber(19.2);
+
+    const featureVersion = await JSON.parse(await getColumnValueFromTableInDB(featuresConfigTable, 'data')).version;
+
+    await t.expect(featureVersion).eql(1, 'Config highest version not applied');
+    await t.expect(browserPage.InsightsPanel.insightsBtn.exists).notOk('Insights panel displayed when disabled in default config');
+});
+test
+    .before(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
```

Comment:
why do we call acceptLicenseTermsAndAddDatabaseApi, if we call it in beforeEach?

Reply:
no, beforeEach applies only if test doesn't have before hook
in this case only before hook applied